### PR TITLE
Add some docs

### PR DESCRIPTION
**Pull request by jurre**
Heya, first of all, thanks for making this, I think it's awesome!

I made a (still very basic) [iOS client](https://github.com/jurre/Mogo-iOS) , and to do so I had to figure out some of the API so I thought I'd document some of that!

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ curl -XGET http://example.com/api/rooms -H 'Authorization: 1n2jjvsns'
 
 Your access to certain API calls depends on the role of your account.
 
+Alternatively you can sign in through the API with your `email` and `password`.
```
Comment (HashNuke) [inline docs/api.md]: Email/password authentication is only for when the session can be maintained - which means it's only useful for browsers and specifically for the web interface. Everything else must use the auth token.

Can you remove this line?

File: docs/api.md [hunk 2]
```diff
@@ -130,7 +134,14 @@ TODO example body
 > POST /messages
```
Comment (HashNuke) [inline docs/api.md:132]: Can you check if this request returns a response? An example response for the example request body mentioned would be nice to have.

File: docs/api.md [hunk 3]
```diff
@@ -172,3 +214,15 @@ Set `joined` to `true` to join a room, or set `false` to leave it.
 ```
 TODO
 ```
+
+## Sessions
+
+#### Sign in
+> POST /api/sessions
```
Comment (HashNuke) [inline docs/api.md:235]: Response body for this request returns an auth token for use with subsequent requests. So showing a response for the request example would be nice.

**Comment by HashNuke** [comment]
Hey @jurre,

Thank you for the PR. This is incredibly useful.

I've added some notes in the comments. Can you please make those changes too? I'll then merge it in.

Also, if you don't mind, get me your twitter handle please :)

**Comment by jurre** [comment]
:+1:  I'll update it in a bit! @jurretweet on twitter btw :)

File: docs/api.md (hunk 2 above)
Comment (jurre) [inline docs/api.md:132]: Yeah it returns the message, I'll update it!

**Comment by jurre** [comment]
@HashNuke done! :)

**Comment by HashNuke** [comment]
Merged ~! Thank you :)

**Pull request merged**